.github/workflows/tests.yml

```diff
@@ -6,21 +6,25 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 4
+      max-parallel: 3
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7]
+        python-version: [2.7, 3.6, 3.7]
 
     steps:
       - uses: actions/checkout@v1
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Get pip cache
+        id: pip-cache
+        run: |
+          python -c "import site; print('::set-output name=dir::' + site.getsitepackages()[0])"
       - uses: actions/cache@v1
         id: cache
         with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/setup.py') }}
+          path: ${{ steps.pip-cache.outputs.dir }}/../../../
```

Comment:
did we figure out why this ends up working? It will be useful to have a comment on why this is a correct path (or a link to some discussions)